runtime/oti/jvminit.h

```diff
@@ -451,6 +451,9 @@ enum INIT_STAGE {
 #define VMOPT_XCOMPRESSEDREFS "-Xcompressedrefs"
 #define VMOPT_XNOCOMPRESSEDREFS "-Xnocompressedrefs"
 
+#define VMOPT_XENABLE_TRANSPARENT_HUGEPAGE "-XenableTransparentHugepage"
+#define VMOPT_XDISABLE_TRANSPARENT_HUGEPAGE "-XdisableTransparentHugepage"
```

Comment:
Please use an `-XX:[+-]TransparentHugePage` option.  XX options are preferable to X ones

Reply:
Updated